shared/src/types/index_set.rs

```diff
@@ -0,0 +1,99 @@
+//! Set data structure optimized to store [`usize`] values.
+
+use std::collections::BTreeMap;
+
+/// The storage unit for the bits in an [`IndexSet`].
+#[allow(dead_code)]
```

Comment:
this module is annotated throughout with `dead_code`, is it replaced by https://github.com/heliaxdev/index-set now?

Reply:
ohhh shit, yeah we should get rid of this!